src/LibVLCSharp.Uno/LibVLCSharp.Uno.csproj

```diff
@@ -5,7 +5,7 @@
     <Summary>Uno integration for LibVLCSharp</Summary>
     <LangVersion>8.0</LangVersion>
     <Nullable>enable</Nullable>
-    <TargetFrameworks>Xamarin.iOS10;MonoAndroid81</TargetFrameworks>
+    <TargetFrameworks>Xamarin.iOS10;MonoAndroid90</TargetFrameworks>
```

Comment:
Would it still work if we only target Android 9 in LibVLCSharp.Uno and remain on Android 8.1 in the other projects?

Reply:
Seems to work.